doc-source/xray-sdk-python-patching.md

```diff
@@ -54,6 +54,38 @@ In some cases, the key that you use to patch a library does not match the librar
 `httplib` – `[httplib](https://docs.python.org/2/library/httplib.html)` and [https://docs.python.org/3/library/http.client.html](https://docs.python.org/3/library/http.client.html)
 `mysql` – `[mysql\-connector\-python](https://pypi.python.org/pypi/mysql-connector-python)`
 
+To patch a single library, call `patch` with a tuple of the library name\. In order to achieve this you will need to add a trailing comma or provide a single element list\.
+
+**Example main\.py – patch single library (trailing comma)**  
+
+```
+import boto3
+import botocore
+import requests
+import mysql-connector-python
+
+from aws_xray_sdk.core import xray_recorder
+from aws_xray_sdk.core import patch
+
+libraries = ('botocore',)
+patch(libraries)
+```
+
+**Example main\.py – patch single library (single element list)**  
+
+```
+import boto3
+import botocore
+import requests
+import mysql-connector-python
+
+from aws_xray_sdk.core import xray_recorder
+from aws_xray_sdk.core import patch
+
+libraries = (['botocore'])
+patch(libraries)
+```
+
```

Comment:
Please remove this example from the PR. We typically only have 1 recommended way of doing something in the documentation, and the other example is more consistent with other patching calls.